plugins/eddn.py

```diff
@@ -487,7 +487,7 @@ def export_journal_entry(self, cmdr: str, is_beta: bool, entry: Mapping[str, Any
 
             if (
                 entry['event'] == 'Docked' or (entry['event'] == 'Location' and entry['Docked']) or not
-                (config.getint('output') & config.OUT_SYS_DELAY)
+                (config.get_int('output') & config.OUT_SYS_DELAY)
```

Comment:
Hmmm, now I wonder if this is the time to abstract these flags.  So that rather than storing them all in one bit-packed DWORD they're actually separate.

But perhaps that's better saved for the great storage method unification.

Reply:
Whoops didnt mean to resolve that. Yeah I think waiting is best. No need to implement double migration code